### ISSUE TITLE
Could not install with yarn

### ISSUE THREAD

**Issue by thebigcosinus** [swiety85/angular2gridster#302]
I can't install it with yarn

```
yarn add v1.5.1
info No lockfile found.
[1/4] Resolving packages...
error An unexpected error occurred: "Couldn't find package \"angular2gridster\" 
```

Seems ok with npm i angular2gridster

thanks

**Comment by sirajc** [comment]
Experiencing the same when we try to install a specific version (6.0.2)
```
yarn add angular2gridster@6.0.2
yarn add v1.9.4
[1/4] 🔍  Resolving packages...
error Couldn't find package "angular2gridster" on the "npm" registry.
info Visit https://yarnpkg.com/en/docs/cli/add for documentation about this command.
```

If you try to install version `6.0.1` then it works perfectly. Upon digging further I found that there are two versions for 6.0.2 in npm registry one is version "6.0.2" published 2 months ago and one is "v6.0.2" which was published yesterday (Oct 4). You might have to remove version "v6.0.2"

cc: @swiety85 @michalstepien

**Comment by sirajc** [comment]
@thebigcosinus You can use version 6.0.1 untill this is resolved `yarn add angular2gridster@6.0.1`

**Comment by swiety85** [comment]
The problem should be solved now. Sorry for inconveniences.
